src/spn/algorithms/MPE.py

```diff
@@ -71,6 +71,8 @@ def get_mpe_top_down_leaf(node, input_vals, data=None, mode=0):
 _node_bottom_up_mpe = {}
 _node_bottom_up_mpe_log = {Sum: sum_log_likelihood, Product: prod_log_likelihood}
 
+def get_node_funtions():
```

Comment:
If this is always of fixed size, maybe we return a tuple instead?

Reply:
Right. I'll have to make that change